src/mmw/js/src/core/views.js

```diff
@@ -188,7 +188,20 @@ var MapView = Marionette.ItemView.extend({
         var map = new L.Map(this.el, {
                 zoomControl: false,
                 attributionControl: options.showLayerAttribution
-            });
+            }),
+            nullVectorLayer = {
+                display: 'nullVector',
+                vector: true,
+                empty: true
+            },
+            nullRasterLayer = {
+                display: 'nullRaster',
+                raster: true,
+                empty: true
+            },
+            vectorLayers = [nullVectorLayer].concat(settings.get('vector_layers')),
+            rasterLayers = [nullRasterLayer].concat(settings.get('raster_layers')),
+            overlayLayers = vectorLayers.concat(rasterLayers);
```

Comment:
Consider splitting this logic out into a separate function that's called in initialize and returns `overayLayers`, just for the sake of keeping the `initialize` function shorter (it was previously ~300 lines).